context/condarc

```diff
@@ -1,7 +1,6 @@
 auto_update_conda: False
 channels:
   - rapidsai-nightly
-  - dask/label/dev
   - pytorch
   - conda-forge
   - nvidia
```

Comment:
Please also remove `nvidia` here. And do we need `pytorch` anymore? They aren't shipping conda packages there anymore.

Reply:
Sure I can expand the scope of this one.